proto/sifnode/oracle/v1/types.proto

```diff
@@ -7,13 +7,47 @@ import "sifnode/oracle/v1/network_descriptor.proto";
 option go_package = "github.com/Sifchain/sifnode/x/oracle/types";
 
 message GenesisState {
-  map<uint32, ValidatorWhiteList> address_whitelist = 1;
+  repeated GenesisValidatorWhiteList validator_whitelist = 1;
   string admin_address = 2;
   repeated Prophecy prophecies = 3;
-  map<uint32, uint32> consensus_needed = 4;
-  map<uint32, CrossChainFeeConfig> cross_chain_fee = 5;
-  map<string, uint64> witness_lock_burn_sequence = 6;
-  map<string, ProphecyInfo> prophecy_info = 7;
+  repeated GenesisConsensusNeeded consensus_needed = 4;
+  repeated GenesisCrossChainFeeConfig cross_chain_fee = 5;
+  repeated GenesisWitnessLockBurnSequence witness_lock_burn_sequence = 6;
+  repeated GenesisProphecyInfo prophecy_info = 7;
+}
+
+message ValidatorWhiteList { 
+  repeated ValidatorPower validator_power = 1;
+}
+
+message ValidatorPower { 
+  bytes validator_address = 1;
+  uint32 voting_power = 2;
+}
+
+message GenesisValidatorWhiteList { 
```

Comment:
These three should be joined into one message type

Reply:
will combine these three items.